src/components/FromSTIX/Statistics.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import styles from "./from_stix.module.scss";
+import {useDispatch, useSelector} from "react-redux";
+import {clearMappings} from "../../store/actions/from_stix";
+import "./Statistics.scss";
+
+
+const Statistics = () => {
+    const dispatch = useDispatch();
+    const mapping = useSelector((state) => state.fromStix.mapping);
+
+
+    function calculateNumber(mapping){
```

Comment:
function name can be more informative if not using comments, "calculateOfficialObjectsCoveragePercentage"